blockchain/state_manager/src/lib.rs

```diff
@@ -734,9 +738,24 @@ where
     /// Performs a state transition, and returns the state and receipt root of
     /// the transition.
     #[instrument(skip(self, callback))]
-    pub async fn compute_tipset_state<CB: 'static>(
+    pub async fn compute_tipset_state_non_blocking<CB: 'static>(
```

Comment:
Any rationale about changing name for `async` version and not using `compute_tipset_state_blocking` for the blocking call?

Not saying it's better (I'm not sure if there are official naming schemes for this), just curious.

Reply:
Hmm, not much thought went into it. I think `compute_tipset_state_blocking` is nicer.